File: despatch.WEB/app/src/swagger/models/user-images-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { ImageResponse } from './image-response';
import { UserResponse } from './user-response';
export interface UserImagesResponse extends BaseEntityResponse{
  image?: ImageResponse;
  imageId?: string;
  user?: UserResponse;
  userId?: string;
}

File: despatch.Domain/Models/Default/Response/UserImagesResponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace despatch.Domain.Models.Default.Response
{
    public class UserImagesResponse : BaseEntityResponse
    {
        public string UserId { get; set; }
        public string ImageId { get; set; }

        public virtual UserResponse User { get; set; }
        public virtual ImageResponse Image { get; set; }
    }
}
